test/test_playbooks/fixtures/location-2.yml

```diff
@@ -1,438 +1,323 @@
-      content-type: [application/json]
+      Accept: [application/json;version=2]
+      Cookie: [_session_id=827f4c1bd93e9698ca43b8be448676e8]
     method: GET
-    uri: https://centos7-katello-nightly.sean.example.com/api/v2/locations
+    uri: https://centos7-foreman-1-22/api/locations?per_page=4294967296&search=title%3D%22Test+Location%2FSub+Location+1%22&thin=False
   response:
-    body:
-      string: !!binary |
-        H4sIAAAAAAAAAz3LsQrCQAzG8f2e4shc0Oom+AZuulkpdzVoIfQkyQ2l9N0be9Utv3/4Juc9aNJA
-        cPLH6ivJ8Rf2a/iEFxrqAuR2C4fyFgzcvc2gvRKeG7ihqL+kLmifht01xz983QCUVWK1zWS3KY52
-        D5moKk78RN6SlXmdMEomFcv3h5vdAmmzgq67AAAA
+    body: {string: !!python/unicode "{\n  \"total\": 2,\n  \"subtotal\": 0,\n  \"page\":
+        1,\n  \"per_page\": 4294967296,\n  \"search\": \"title=\\\"Test Location/Sub
+        Location 1\\\"\",\n  \"sort\": {\n    \"by\": null,\n    \"order\": null\n
+        \ },\n  \"results\": []\n}\n"}
     headers:
-      apipie-checksum: [fb425d67305247df38ecf5e4c6714214eb9c24aa]
+      apipie-checksum: [9f3fe04d3a1b506370bbd7924e81c1ffa9070904]
       cache-control: ['max-age=0, private, must-revalidate']
-      connection: [Keep-Alive]
-      content-encoding: [gzip]
-      content-length: ['144']
       content-security-policy: ['default-src ''self''; child-src ''self''; connect-src
           ''self'' ws: wss:; img-src ''self'' data: *.gravatar.com; script-src ''unsafe-eval''
           ''unsafe-inline'' ''self''; style-src ''unsafe-inline'' ''self''']
       content-type: [application/json; charset=utf-8]
-      date: ['Tue, 15 Jan 2019 09:57:41 GMT']
-      etag: [W/"ac4f79659e80702c1440b8a061e9db54-gzip"]
+      date: ['Tue, 04 Jun 2019 20:05:42 GMT']
+      etag: [W/"b8e71e03e170fd82412b76da69f794cd"]
       foreman_api_version: ['2']
-      foreman_version: [1.20.0-develop]
-      keep-alive: ['timeout=5, max=10000']
-      server: [Apache/2.4.6 (CentOS)]
-      set-cookie: [_session_id=eb2ec9624b2083bfe2cc2ccba2fec5b1; path=/; secure; HttpOnly;
-          SameSite=Lax]
+      foreman_version: [1.22.0-RC2]
+      server: [Apache]
       status: [200 OK]
       strict-transport-security: [max-age=631139040; includeSubdomains]
+      transfer-encoding: [chunked]
       vary: [Accept-Encoding]
       x-content-type-options: [nosniff]
       x-download-options: [noopen]
       x-frame-options: [sameorigin]
       x-permitted-cross-domain-policies: [none]
       x-powered-by: [Phusion Passenger 4.0.53]
-      x-request-id: [d80ea1bc-fa7b-4418-84a1-61434b90d79a]
-      x-runtime: ['0.050843']
+      x-request-id: [df7ba823-b208-4e98-94bb-e4e8c077343a]
+      x-runtime: ['0.011515']
       x-xss-protection: [1; mode=block]
     status: {code: 200, message: OK}
 - request:
-    body: !!python/unicode '{"location": {"parent_id": 35, "organization_ids": [28],
-      "name": "Sub Location 1"}}'
+    body: !!python/unicode '{"location": {"parent_id": 33, "name": "Sub Location 1"}}'
```

Comment:
This was supposed to contain the organization id.

`{'parent_id': 33, 'name': 'Sub Location 1', 'organization_ids': [30]}` is what gets passed to `self._resource_action(resource, 'create', new_entity)`.
@evgeni any idea?